contextual_services/contextual_services_sd.model.lkml

```diff
@@ -0,0 +1,4 @@
+connection: "telemetry"
+label: "Contextual Services on spoke-default"
+include: "//looker-hub/contextual_services/views/*"
+include: "explores/*"
```

Comment:
The issue here is that the model name was changed. The model file should have the same name as the namespace folder

Reply:
Ah, I see. There is a `contextual_services` namespace on spoke-private

Reply:
disregard previous comment